File: 7_Kyu_Kata/categorize_new_member/JavaScript/categorizeNewMember.js
// 7 kyu - Categorize New Member  [ ID: 5502c9e7b3216ec63c0001aa  (categorize-new-member) ]
// URL: https://www.codewars.com/kata/5502c9e7b3216ec63c0001aa
// Category: undefined  |  Tags: FUNDAMENTALS
// *****************************************************************************
const openOrSenior = (data) => data.map(([a, h])=> a >= 55 && h > 7 ? "Senior" : "Open")
// *****************************************************************************
// *****************************************************************************
const openOrSenior = data => data.map(x => x[0] > 54 && x[1] > 7 ? "Senior" : "Open")

File: 7-kyu-kata/categorize-new-member/python/categorize_new_member.py
# 7 kyu - Categorize New Member  [ ID: 5502c9e7b3216ec63c0001aa  (categorize-new-member) ]
# URL: https://www.codewars.com/kata/5502c9e7b3216ec63c0001aa
# Category: REFERENCE  |  Tags: FUNDAMENTALS
# ******************************************************************************
def open_or_senior(data):
    return ["Senior" if a >= 55 and h > 7 else "Open" for a, h in data]
